CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 2.1.1 – 2022-11-14
+## Added
```

Comment:
```suggestion
### Added
```

``##`` = h2
``###`` = h3

Reply:
On that note. This is a patch release. Why is there a new feature?!

Reply:
https://github.com/nextcloud/mail/pull/7569 was mislabeled. It's a fix.

Reply:
It's a *fix*

Reply:
i actually added it as a changed in the other log(1.14) since it doesnt break anything but it can look better

Reply:
It was a design bug. Move it into *Fixed* please.

Reply:
ok, so patches cannot have changed or added features: https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html#versioning